compiler/rustc_builtin_macros/src/lib.rs

```diff
@@ -76,6 +77,8 @@ pub fn register_builtin_macros(resolver: &mut dyn ResolverExpand) {
         log_syntax: log_syntax::expand_log_syntax,
         module_path: source_util::expand_mod,
         option_env: env::expand_option_env,
+        core_panic_macro: panic::expand_panic,
+        std_panic_macro: panic::expand_panic,
```

Comment:
```suggestion
        core_panic: panic::expand_panic,
        std_panic: panic::expand_panic,
```
(To avoid tautology and be more consistent with other macros.)